### PR TITLE
Synapse 1.67

### PR DESCRIPTION
**Pull request by babolivier**
Closes https://github.com/matrix-org/synapse/issues/13744

Preview: https://pr1441--matrix-org-previews.netlify.app

### CODE REVIEW

File: gatsby/content/blog/2022/09/2022-09-13-synapse-1.67.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+protocol is now the only protocol that can be used.
+
+See the [upgrade
+notes](https://matrix-org.github.io/synapse/v1.67/upgrade.html#direct-tcp-replication-is-no-longer-supported-migrate-to-redis)
+for more information.
+
+## Changes for installations using a source checkout
+
+Server admins who installed Synapse using a source checkout should be familiar
+with [Poetry](https://python-poetry.org/) by now. This is a tool we use to
```
Comment (reivilibre) [inline gatsby/content/blog/2022/09/2022-09-13-synapse-1.67.mdx:36]: not entirely true; you can still use pip and forget about Poetry. (my homeserver still does this)

File: gatsby/content/blog/2022/09/2022-09-13-synapse-1.67.mdx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+It is also worth noting that Synapse 1.68, which is due to release around
+September 27, will require a Rust compiler when installed from a source
+checkout. This is because we are in the process of introducing Rust in Synapse's
+code base, in order to make some hot code paths more efficient. See the [upgrade
+notes](https://matrix-org.github.io/synapse/v1.67/upgrade.html#rust-requirement-in-the-next-release)
+for more information.
+
+These two changes should not impact installations that use `pip install
+matrix-synapse`, distribution-specific packages or Docker images to manage
```
Comment (reivilibre) [inline gatsby/content/blog/2022/09/2022-09-13-synapse-1.67.mdx:49]: not entirely true; less common & non-Linux platforms will still need a Rust compiler to install from PyPI

**Review by reivilibre — APPROVED** [review]
lgtm overall; just two minor points

File: gatsby/content/blog/2022/09/2022-09-13-synapse-1.67.mdx (hunk 1 above)
Comment (babolivier) [reply gatsby/content/blog/2022/09/2022-09-13-synapse-1.67.mdx:36]: Good point.

**Comment by DMRobertson** [comment]
Closes matrix-org/synapse#13744

**Pull request merged**